pyresample/ewa/__init__.py

```diff
@@ -117,8 +117,8 @@ def ll2cr(swath_def, area_def, fill=np.nan, copy=True):
     ch = -abs(area_def.pixel_size_y)
     w = area_def.x_size
     h = area_def.y_size
-    ox = area_def.area_extent[0]
-    oy = area_def.area_extent[3]
+    ox = area_def.area_extent[0] + cw / 2.
```

Comment:
isn't `area_extent[0]` the lower left x coordinate ? shouldn't it be minus then ?